<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstPage</class>
 <widget class="QWidget" name="FirstPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>기본정보 등록</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="to_mainButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
     </font>
    </property>
    <property name="text">
     <string>돌아가기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
     </font>
    </property>
    <property name="text">
     <string>저장</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>600</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>중앙대학교 포탈 로그인 ⇒ 강의마당 ⇒ 성적 ⇒ 보관성적조회 ⇒ 성적내역 복사 &amp; 붙여넣기</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>600</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>180</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>학부(과)/전공 선택</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>국어국문학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>독일어문학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>러시아어문학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>역사학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>영어영문학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>일본어문학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>중국어문학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>철학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>프랑스어문학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>공공인재학부</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>도시계획부동산학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>디지털미디어콘텐츠전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>문헌정보학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>사회복지전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>사회학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>심리학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>언론정보전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>정치국제학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>교육학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>영어교육과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>유아교육과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>체육교육과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>물리학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>생명과학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>수학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>화학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>건설환경플랜트공학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>건축공학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>건축학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>기계공학부</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>도시시스템공학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>발전기계전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>발전전기전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>원자력전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>첨단소재공학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>화학공학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>나노소재공학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>바이오메디컬공학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>전자전기공학부</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>소프트웨어학부</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AI학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>경영학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>경제학부</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>광고홍보학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>국제물류학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>글로벌금융전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>산업보안학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>응용통계학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>지식경영학부</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>의학부</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>약학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>제약학전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>간호학과</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>공간연출전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>연극전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>영화전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>게임콘텐츠애니메이션전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>공예전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>관현악전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>무용전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>문예창작전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>사진전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>산업디자인전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>서양화전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>성악전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>시각디자인전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>실내환경디자인전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>실용음악전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>연희예술전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>음악예술전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>작곡전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>조소전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>패션전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>피아노전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>한국화전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TV방송연예전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>예술공학부</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>생활 레저스포츠전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>스포츠산업전공</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>골프전공</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>180</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>단과대학 선택</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>인문대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>사회과학대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>사범대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>자연과학대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>공과대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>창의ICT공과대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>소프트웨어대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>경영경제대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>약학대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>의과대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>적십자간호대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>예술대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>체육대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>생명공학대학</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>예술공학대학</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>2025</string>
    </property>
    <item>
     <property name="text">
      <string>2025</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2023</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2022</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2021</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2020</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2018</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2017</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2016</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2015</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2014</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2013</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2012</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2011</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2010</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2009</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2008</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2007</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2006</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2005</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2004</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2003</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2002</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1999</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1998</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1997</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>85</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>입학연도</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
